cmd/thriftbreak/main.go

```diff
@@ -57,8 +59,18 @@ func run(args []string) error {
 	}
 
 	lints := pass.Lints()
+	var out string
 	for _, l := range lints {
-		fmt.Println(l.String())
+		if *jsonOut {
```

Comment:
If we do this, aren't we appending many JSONs, separated by a newline? Sounds like that wouldn't be a valid JSON format if you piped the output - is that what we want or should we output one valid JSON-formatted string instead?

Reply:
I think the typical behavior for something like this is newline delimited JSON objects. These can be fed directly into jq.

Reply:
(A single JSON string can also be fed into jq, but there's explicit support for ndjson in jq.)

Reply:
i see. nvm then :)